packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js

```diff
@@ -9,61 +9,106 @@
 
 'use strict';
 
-var SyntheticWheelEvent;
-
 describe('SyntheticWheelEvent', () => {
-  var createEvent;
+  var React;
+  var ReactDOM;
+  var createMouseEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticWheelEvent = require('../SyntheticWheelEvent').default;
-
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget').default(nativeEvent);
-      return SyntheticWheelEvent.getPooled({}, '', nativeEvent, target);
+    React = require('react');
+    ReactDOM = require('react-dom');
+    createMouseEvent = (type = '', options = {}) => {
+      const event = document.createEvent('MouseEvent');
+      event.initEvent(type, true, false);
+      return Object.assign(event, options);
     };
   });
 
   it('should normalize properties from the Event interface', () => {
-    var target = document.createElement('div');
-    var syntheticEvent = createEvent({srcElement: target});
+    const container = document.createElement('div');
+    const event = createMouseEvent('', {srcElement: container});
+    container.dispatchEvent(event);
 
-    expect(syntheticEvent.target).toBe(target);
-    expect(syntheticEvent.type).toBe(undefined);
+    expect(event.target).toBe(container);
+    expect(event.type).toBe('');
   });
 
   it('should normalize properties from the MouseEvent interface', () => {
-    expect(createEvent({which: 2, button: 1}).button).toBe(1);
+    const event = new MouseEvent('', {
+      bubbles: true,
+      cancelable: false,
+      which: 2,
+      button: 1,
+    });
+    expect(event.button).toBe(1);
```

Comment:
This doesn't test the React's synthetic event at all. It just creates a jsdom event object and then asserts that object works. This is not a useful test for us because we already know jsdom works (it has its own test suite).

We should instead test that the synthetic event object produced by React has the right fields.